[2025-05-08T09:06:16.714+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: final_project_BBDL.LandingToBronze manual__2025-05-08T09:06:14.604814+00:00 [queued]>
[2025-05-08T09:06:16.725+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: final_project_BBDL.LandingToBronze manual__2025-05-08T09:06:14.604814+00:00 [queued]>
[2025-05-08T09:06:16.726+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2025-05-08T09:06:16.768+0000] {taskinstance.py:1327} INFO - Executing <Task(SparkSubmitOperator): LandingToBronze> on 2025-05-08 09:06:14.604814+00:00
[2025-05-08T09:06:16.771+0000] {standard_task_runner.py:57} INFO - Started process 1193 to run task
[2025-05-08T09:06:16.774+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'final_project_BBDL', 'LandingToBronze', 'manual__2025-05-08T09:06:14.604814+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/project_solution.py', '--cfg-path', '/tmp/tmpckr6oydd']
[2025-05-08T09:06:16.776+0000] {standard_task_runner.py:85} INFO - Job 13: Subtask LandingToBronze
[2025-05-08T09:06:16.833+0000] {logging_mixin.py:150} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-08T09:06:16.875+0000] {task_command.py:410} INFO - Running <TaskInstance: final_project_BBDL.LandingToBronze manual__2025-05-08T09:06:14.604814+00:00 [running]> on host 6d297fae3d14
[2025-05-08T09:06:17.051+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='final_project_BBDL' AIRFLOW_CTX_TASK_ID='LandingToBronze' AIRFLOW_CTX_EXECUTION_DATE='2025-05-08T09:06:14.604814+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-08T09:06:14.604814+00:00'
[2025-05-08T09:06:17.062+0000] {base.py:73} INFO - Using connection ID 'spark_default' for task execution.
[2025-05-08T09:06:17.064+0000] {spark_submit.py:339} INFO - Spark-Submit cmd: spark-submit --master spark://spark-master:7077 --conf spark.master=spark://spark-master:7077 --name LandingToBronze --verbose --queue root.default /opt/airflow/dags/landing_to_bronze.py
[2025-05-08T09:06:17.079+0000] {spark_submit.py:490} INFO - /opt/spark/bin/spark-class: line 71: /usr/lib/jvm/java-11-openjdk-amd64/bin/java: No such file or directory
[2025-05-08T09:06:17.079+0000] {spark_submit.py:490} INFO - /opt/spark/bin/spark-class: line 96: CMD: bad array subscript
[2025-05-08T09:06:17.095+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 157, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 422, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark-master:7077 --conf spark.master=spark://spark-master:7077 --name LandingToBronze --verbose --queue root.default /opt/airflow/dags/landing_to_bronze.py. Error code is: 1.
[2025-05-08T09:06:17.102+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=final_project_BBDL, task_id=LandingToBronze, execution_date=20250508T090614, start_date=20250508T090616, end_date=20250508T090617
[2025-05-08T09:06:17.137+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 13 for task LandingToBronze (Cannot execute: spark-submit --master spark://spark-master:7077 --conf spark.master=spark://spark-master:7077 --name LandingToBronze --verbose --queue root.default /opt/airflow/dags/landing_to_bronze.py. Error code is: 1.; 1193)
[2025-05-08T09:06:17.175+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2025-05-08T09:06:17.202+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
